barretenberg/cpp/src/barretenberg/honk/utils/honk_key_gen.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
```

Comment:
I have thrown this in the honk/utils path as it seemed fitting, however this path only has testing code. @codygunton do you think this should stay here?

Reply:
Another file like this is in barretenberg/solidity_helpers but idk exactly what makes sense